app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,69 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = current_user.foods
+  end
+
```

Comment:
- [ ] Let's improve upon the performance of our queries by including solutions to `N + 1` problem.

Reply:
Hello @rloterh. This list is just the current user for the food render. I don't think it will cause an N +1 issue. Seeing that the foods for a special log-in user render as an index

Reply:
@OscarFMdev 
Here again, you don't necessary have to implemented the said changes in the `foods controller`. Solving N + 1 most of the time will takes place in the controller, where the queries are used in the controller actions needs to be optimized. Please figure out the appropriate controller(s) of which the actions will need the `N+1 optimization.

Reply:
Well resolved ✔️